### PR TITLE
⬆️ 1.20.6 Support

### PR DESCRIPTION
**Pull request by GreenScreen410**

### CODE REVIEW

**Comment by GreenScreen410** [comment]
Only tested in bukkit.

**Comment by wacch** [comment]
Tested on Purpur 1.20.6 (using FAWE and WE)

**Review by wacch — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -4,5 +4,5 @@ plugins {
 
 allprojects {
     group = "me.kous500.curvebuilding"
-    version = "0.6.0"
+    version = "0.7.0"
```
Comment (wacch) [inline build.gradle:7]:
```suggestion
    version = "0.6.1"
```

**Comment by wacch** [comment]
@GreenScreen410 
Please change the plugin version from `0.7.0` to `0.6.1`.

**Pull request closed without merging**